src/components/toast/toast.ts

```diff
@@ -188,7 +188,7 @@ class ToastCmp {
     }
   }
 
-  ionViewDidEnter() {
+  ngAfterViewInit() {
```

Comment:
Putting the focus in AfterViewInit may be too soon before the element can receive focus in the DOM.